### PR TITLE
Added get geek quote service

### PR DESCRIPTION
**Pull request by santiago-b9826**
## Summary

I consumed [geek joke api](https://geek-jokes.sameerkumar.website/api?format=json) to generate random quotes. 
I Added corresponding errors to the requested API.

## Known Issues
None

## Trello Card

https://trello.com/c/JgRR27w6/32-servicios-promises-witter

### CODE REVIEW

File: app/controllers/weets.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const logger = require('../logger');
```
Comment (jonnyalexbh) [inline app/controllers/weets.js]: good job!
But i think this card is about services only

File: app/routes.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { getQuote } = require('./controllers/weets');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
+  app.get('/weets', getQuote);
   // app.put('/endpoint/put/path', [], controller.methodPUT);
```
Comment (jonnyalexbh) [inline app/routes.js]: You can delete the comments :)

File: config/axios.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
```
Comment (jonnyalexbh) [inline config/axios.js]: I think it would be better in a folder of helpers, what do you think?

File: config/index.js [hunk 4]
```diff
@@ -45,6 +45,11 @@ const config = {
       packageVersion: process.env.PACKAGE_VERSION || 'X-Package-Version',
       nodeVersion: process.env.NODE_VERSION || 'X-Node-Version'
     }
+  },
+  external: {
+    geekJokeApi: {
+      url: process.env.GEEK_JOKE_URL
```
Comment (jonnyalexbh) [inline config/index.js:51]: it would be nice to add this environment variable GEEK_JOKE_URL in the .env.example file

File: app/routes.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { getQuote } = require('./controllers/weets');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
+  app.get('/weets', getQuote);
```
Comment (jonnyalexbh) [inline app/routes.js]: I think this card is about services only

File: app/services/index.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+const weets = require('./weets');
```
Comment (jonnyalexbh) [inline app/services/index.js:1]: delete the .keep file

File: config/axios.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+const config = require('../config').external.geekJokeApi;
+
+const geekJoke = axios.create({
+  baseURL: config.url,
```
Comment (jonnyalexbh) [inline config/axios.js]: This helper would leave it a little more generic, think about it in the future

File: app/routes.js (hunk 5 above)
Comment (santiago-b9826) [reply app/routes.js]: Should I remove the controller and route? Or keep this and be more careful in future work

File: config/axios.js (hunk 3 above)
Comment (santiago-b9826) [reply config/axios.js]: It is true, reading the description of the helpers in the tech guide I see that it meets the main characteristics, does not contain business logic and could be implemented in other projects.

File: app/routes.js (hunk 5 above)
Comment (jonnyalexbh) [reply app/routes.js]: should remove both of them, use the console created by Wolox to test it

File: app/routes.js [hunk 8]
```diff
@@ -3,7 +3,4 @@ const { healthCheck } = require('./controllers/healthCheck');
```
Comment (jonnyalexbh) [inline app/routes.js:3]: remove this`// const controller = require('./controllers/controller');`

**Review by jonnyalexbh — APPROVED** [review]

**Pull request merged**